overview/analytics.md

```diff
@@ -50,30 +50,112 @@ pbjs.que.push(function() {
 
 {% assign analytics_pages = site.pages | where: "layout", "analytics" %}
 
-<ul>
+## Search for analytics adapters
+
+<input type="text" id="autocomplete-filter" class="autocomplete-filter">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/awesomplete/1.1.5/awesomplete.min.js" integrity="sha512-HcBl0GSJvt4Qecm4srHapirUx0HJDi2zYXm6KUKNNUGdTIN9cBwakVZHWmRVj4MKgy1AChqhWGYcMDbRKgO0zg==" crossorigin="anonymous"></script>
+<script>
+var AutocompleteList = [{% for page in analytics_pages %}{ label: '{{ page.title }}', value: '{{ page.url }}' },{% endfor %}];
```

Comment:
These are the things that are amazing about Jekyll & Liquid. I really think this is nice